CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 
 
+## 3.2.0 - 2021-05-26
```

Comment:
This shouldn't be a major version. You could argue for a minor or patch version.

Reply:
Ar first I was only allowing php 8 and that's why I set that version number.

Now a make it a minor.